### DISCUSSION TITLE
Can I export to HTML with this plugin?

### DISCUSSION THREAD

**Discussion by dionebigode** [ephread/inkgd#87] [Q&A] [answered]
I've been trying to export the example to HTML5, but it just won't work. 

When I pick one of the stories, it just hangs while loading without outputting any sort of error. 

I'm a newby on this matter, so I think I might have my head up my arse and am trying to do something that is obviously not possible.

**Comment by francoisdlt** [comment, marked as answer]
Have you added the json to the exported file formats ? By default it wont be exported

**Reply by dionebigode** [reply]
So, here's my process to get a 'hello world' working:
First, I worked the examples into a main scene and managed to get it working 
![image](https://github.com/user-attachments/assets/dccb5ab0-edc5-419b-9b0d-d36f87d41c29)

On the export project, under resources, I made sure to add the json and ink files
![image](https://github.com/user-attachments/assets/9a377c87-7088-4837-9140-6c3bd3fd0e8c)

Then I've set up a https server via Python
![image](https://github.com/user-attachments/assets/0961c73b-c5c9-4a26-bd76-fbd713023b7a)

AND IT WORKED!!!
![image](https://github.com/user-attachments/assets/d30612a1-df7b-4884-b762-34b3ffdcf9e0)

Dang, @francoisdlt, you're the best!! I think I probably forgot to add the json on the resources panel!
